permissions/static/js/team.js

```diff
@@ -0,0 +1,42 @@
+const courseSelectField = $("#courses");
+const courseFormGroup = $(".form-group").eq(3);
+
+courseSelectField.change(function(e) {
+  const { value } = e.target;
+  const data = { course_id: value };
+  console.log(value);
+
+  $.ajax({
+    data,
+    url: "/permissions/units/get/",
+    success: function(result) {
+      addModuleOptions(result);
+    }
+  });
+});
+
+function addModuleOptions({ units }) {
+  let html = `
+    <div class="form-group" id="unit-group">

```

Comment:
Instead of using direct HTML tags in JS. You can use document.createElement(). For e.g document.createElement("div");

Reply:
This can be done, but I feel this makes the code more verbose without any benefits. The current method is pretty readable and easy to modify.

Reply:
Instead of you making template in JS. You can create a template and load the data in the template in views and send to your main template.
Using html tags directly is not a cleaner way of doing.